httpx_ws/_api.py

```diff
@@ -83,12 +82,25 @@ def __init__(
 
         self._ping_manager = PingManager()
         self._should_close = threading.Event()
+        self._should_close_task: typing.Optional[concurrent.futures.Future[bool]] = None
+        self._executor: typing.Optional[concurrent.futures.ThreadPoolExecutor] = None
 
         self._max_message_size_bytes = max_message_size_bytes
         self._queue_size = queue_size
         self._keepalive_ping_interval_seconds = keepalive_ping_interval_seconds
         self._keepalive_ping_timeout_seconds = keepalive_ping_timeout_seconds
 
+    def _get_executor_should_close_task(
+        self,
+    ) -> typing.Tuple[
+        concurrent.futures.ThreadPoolExecutor, concurrent.futures.Future[bool]
```

Comment:
`concurrent.futures.Future[bool]` provokes an error in Python 3.8 (don't really understand why), just writing `concurrent.futures.Future` makes it work.

Reply:
Actually adding quotes is fine too: `"concurrent.futures.Future[bool]"`.